pl_examples/hydra_examples/conf/data/kmnist.yaml

```diff
@@ -0,0 +1,32 @@
+# @package _group_
+
+ds:
+  cls: torchvision.datasets.FashionMNIST
+  params:
+    root: ${hydra:runtime.cwd}/datasets
+    download: true
+    train: null
+
+tf:
+  - tensor:
+    cls: torchvision.transforms.ToTensor
+  - normal:
+    cls: torchvision.transforms.Normalize
+    params:
+      mean: .5
+      std: .2
+
+dl:
+  cls: torch.utils.data.DataLoader
+  params:
+    batch_size: 5
+    shuffle: true
+    num_workers: 4
+    pin_memory: False
+    drop_last: False
+    timeout: 0
+    #sampler: None  TODO
```

Comment:
What do you think about the dataset configuration? At the moment, we are missing some of the functionality of a dataloader since each of these are further callable objects. Should I go ahead and create a better version or leave this up to the user?

Reply:
I think leave it as is. you can also demonstrate in how to passthrough stuff.
```python
instantiate(cfg.data, sampler=Sampler(1,2,3))
```

Reply:
since you already have passthrough:

I think we should eventually have those. if it's not too much work you can support it now.
otherwise it can be left for when someone integrates this into PT.

Reply:
Actually the TODO settings are not recommended to be set in PL, PL replaces your sampler and deals with multiprocessing. So I am going to remove these.  https://pytorch-lightning.readthedocs.io/en/latest/introduction_guide.html#tpus